client/blocks/upwork-banner/style.scss

```diff
@@ -0,0 +1,79 @@
+.upwork-banner {
+	background: no-repeat var( --color-accent );
+	display: block;
+	overflow: hidden;
+	padding: 0.8em;
+	position: relative;
+	user-select: none;
+
+	h1,
+	p,
+	.button.upwork-banner__cta {
+		color: var( --color-neutral-700 );
+		position: relative;
+		z-index: z-index( '.upwork-banner', '.button.upwork-banner__cta' );
+	}
+
+	h1 {
+		font-size: 1.6em;
+		font-weight: 500;
+		letter-spacing: 0.01em;
+		margin: -0.1em 0;
+		line-height: 1.3;
+
+		+ p {
+			margin-top: 0.6em;
+		}
+	}
+
+	p {
+		margin: 0;
+	}
+
+	img {
+		transform-origin: top right;
+		max-width: 190px;
+		transition: opacity 400ms ease-in-out;
+		position: absolute;
+		top: 0;
+		right: 30px;
+		user-select: none;
+
+		@include breakpoint( '<1400px' ) {
+			opacity: 0.5;
+		}
+
+		@include breakpoint( '<960px' ) {
+			display: none;
+		}
+	}
+
+	.button.upwork-banner__cta {
+		color: white;
+		margin-top: 1em;
+	}
+
+	.button.upwork-banner__close {
+		display: block;
+		background: rgba( white, 0.66 );
+		border: none;
+		border-radius: 0;
+		line-height: 0.9;
+		padding: 1px 0;
+		position: absolute;
+		right: 0;
```

Comment:
I think this `right` and `top` need to be indented.

Reply:
done